lang/java/reef-runtime-azbatch/src/main/java/org/apache/reef/runime/azbatch/client/AzureBatchRuntimeConfiguration.java

```diff
@@ -19,12 +19,13 @@
 package org.apache.reef.runime.azbatch.client;
 
 import org.apache.reef.annotations.audience.Public;
-import org.apache.reef.runime.azbatch.parameters.AzureBatchAccountKey;
-import org.apache.reef.runime.azbatch.parameters.AzureBatchAccountName;
-import org.apache.reef.runime.azbatch.parameters.AzureBatchAccountUri;
-import org.apache.reef.runime.azbatch.parameters.AzureBatchPoolId;
+import org.apache.reef.runime.azbatch.parameters.*;
+import org.apache.reef.runtime.hdinsight.parameters.AzureStorageAccountContainerName;
```

Comment:
same about the dependency on hdinsight. We have want to move it to common or even duplicate?

Reply:
Same as above.